sci-physics/cernlib/cernlib-2023.10.31.0.ebuild

```diff
@@ -0,0 +1,34 @@
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake
+
+DESCRIPTION="CERNLIB - CERN's library for data analysis"
+HOMEPAGE="https://cernlib.web.cern.ch/cernlib/"
+SRC_URI="https://cernlib.web.cern.ch/download/2023_source/tar/${P}-free.tar.gz"
+S="${WORKDIR}/${P}-free"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	x11-libs/motif:0
+	virtual/lapack
+	dev-lang/cfortran
+	x11-libs/xbae
+	net-libs/libnsl
+"
+DEPEND="
+	${RDEPEND}
+	x11-misc/imake
+	x11-misc/makedepend
+	virtual/pkgconfig
+"
+
+src_install() {
+	cmake_src_install
+	# do not install cfortran.h, it conflicts with ROOT
```

Comment:
Does it make sense to move this to a subdir instead? Or are these headers completely unused?

Reply:
I followed the https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=cernlib but had to remove the second one too.

Reply:
The headers are actually the same (up to version differences), so I removed it in `src_prepare` now already, since it exists already from `dev-lang/cfortran`.

Reply:
Great, Thanks :+1: 

I think we should add a little comment here though, because this is the kind of thing that in x years time is going to be confusing again.